### DISCUSSION TITLE
Creating objects with multiple level of hierarchy

### DISCUSSION THREAD

**Discussion by nkumart** [ozlerhakan/poiji#198] [Q&A]
Does this library supports mapping excel data to object which has multiple level of hierarchy in it ? if it supports how to specify attributes in class ? 

I tried creating a class like below but it is not mapping the values to all attributes for base(here it is EquipmentPoiModel) class.
![image](https://user-images.githubusercontent.com/6207964/111775806-38f72800-88d7-11eb-8da9-ed2e09d78579.png)

Any suggestion would be a great help.
Thanks

**Comment by ozlerhakan** [comment]
Hi @nkumart , Poiji does not support this option, what do you think of enable it with the next release? :)
